### PR TITLE
BaSP-TG-42: Redux - Super Admin

### PR DESCRIPTION
**Pull request by mgriolfo-g**
Implementation of redux over the Super Admin features. Based on Admin Redux and other implementations, developed cooperatively over the past couple of days!

### CODE REVIEW

**Review by JuliBalcaza — APPROVED** [review]
It's nice that you added uppercases

**Comment by JuliBalcaza** [comment]
approved by QA, as a suggestion remember to add a message to the list when it is empty ("Empty list" or "No Super Admins Yet").

**Comment by mgriolfo-g** [comment]
> approved by QA, as a suggestion remember to add a message to the list when it is empty ("Empty list" or "No Super Admins Yet").

Thanks for the suggestion! Pushed changes that made that state unreachable, it should appear on Admins and Super Admins now! Please check it again 😄

File: src/redux/superAdmins/thunks.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      const response = await fetch(`${process.env.REACT_APP_API_URL}/super-admins`, {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify(superAdmin)
+      });
+      const data = await response.json();
+      if (response.ok) {
+        dispatch(postSuperAdminsSuccess(data.data));
+      } else {
+        console.log(data);
```
Comment (juliancit0) [inline src/redux/superAdmins/thunks.js:65]:
```suggestion
```
You should delete this console.log()

File: src/redux/superAdmins/thunks.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      const response = await fetch(`${process.env.REACT_APP_API_URL}/super-admins/${id}`, {
+        method: 'PUT',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify(superAdmin)
+      });
+      const data = await response.json();
+      if (response.ok) {
+        dispatch(putSuperAdminsSuccess(data.data));
+      } else {
+        console.log(data);
```
Comment (juliancit0) [inline src/redux/superAdmins/thunks.js:87]:
```suggestion
```
And this too

**Review by juliancit0 — CHANGES_REQUESTED** [review]

File: src/redux/superAdmins/thunks.js (hunk 2 above)
Comment (mgriolfo-g) [reply src/redux/superAdmins/thunks.js]: Done, thanks again 😄

**Review by juliancit0 — APPROVED** [review]

**Review by TomasBettini — APPROVED** [review]
Good job

**Pull request merged**